src/Widgets/VerticalNavigationWidget/VerticalNavigationWidgetComponent.tsx

```diff
@@ -84,40 +84,32 @@ const SubNavItems = connect(
 
 const ExpandableNavItem = connect(
   ({ obj, navigationDepth }: { obj: Obj; navigationDepth: number }) => {
-    const [isExpanded, setIsExpanded] = useState(false)
+    if (obj.get('hideInNavigation') === true) return null
+
     const children = obj
       .orderedChildren()
       .filter((c) => c.get('hideInNavigation') !== true)
-
-    const hasChildren = children.length > 0
     const isActive = isOnCurrentPath(obj)
-    const isInitiallyExpanded = hasChildren && isActive && !isCurrentPage(obj)
-
-    useEffect(() => {
-      if (isInitiallyExpanded) setIsExpanded(true)
-    }, [isInitiallyExpanded])
-
-    if (obj.get('hideInNavigation') === true) return null
 
-    if (!hasChildren) return <NavItem obj={obj} isActive={isActive} />
+    if (children.length === 0) return <NavItem obj={obj} isActive={isActive} />
 
     const key = `VerticalNavigationWidget-expandable-${obj.id()}`
 
     return (
       <li className={isActive ? 'active' : ''}>
         <Nav.Link as={LinkTag} eventKey={key} key={key} to={obj}>
           <button
-            className={`dropdown-toggle nav-link${isExpanded ? ' show' : ''}`}
+            className={`dropdown-toggle nav-link${isActive ? ' show' : ''}`}
             onClick={(e) => {
               e.preventDefault()
               e.stopPropagation()
-              setIsExpanded((isExpanded) => !isExpanded)
+              navigateTo(obj, { hash: '›' })
             }}
```

Comment:
I don't think that the `onClick` is needed anymore. I would simply remove it.

Reply:
If we remove this, a click on the toggle button would behave exactly like a click on the outer element:
* close the nav b/c of `collapseOnSelect`
* scroll to the top of the page b/c of the default `LinkTag`/`navigateTo` implementation
 
A mobile visitor must reopen the menu to reach a child item.
All visitors (but it's more likely on mobile) may have to scroll down to reach a child item.

I'm fine with one or both of these consequences. WDYT?

Reply:
Yes, please remove this. Maybe also remove the button and make it a simple span.